DRAFT.md

```diff
@@ -1497,108 +1472,111 @@ constexpr polymorphic(allocator_arg_t, const Allocator& a,
   pointers.]_
 
 ```c++
-template <class U, class... Ts>
-explicit constexpr polymorphic(in_place_type_t<U>, Ts&&... ts);
+template <class U=T>
+explicit constexpr polymorphic(U&& u);
 ```
 
-17. _Constraints_: `is_base_of_v<T, U>` is `true`. `is_constructible_v<U,
-  Ts...>` is `true`. `is_copy_constructible_v<U>` is `true`.
-  `is_default_constructible_v<allocator_type>` is `true`.
+11. _Constraints_: `is_same_v<remove_cv_ref_t<U>, U>`.
+   `is_same_v<U, polymorphic>` is `false`.
+   `derived_from<U, T>` is `true`.
+   `is_copy_constructible_v<U>` is `true`.
+   `is_constructible_v<U, U>` is `true`.
+   `U` is not a specialization of `in_place_type_t`.
```

Comment:
```suggestion
11. _Constraints_:  Let `UR` be `remove_cv_ref_t<U>`.
   `is_same_v<UR, polymorphic>` is `false`.
   `derived_from<UR, T>` is `true`.
   `is_copy_constructible_v<UR>` is `true`.
   `is_constructible_v<UR, U>` is `true`.
   `UR` is not a specialization of `in_place_type_t`.
```

(but wants some "and" added between conditions, as discussed on mattermost between Jon and Jon)